python/pyspark/sql/column.pyi

```diff
@@ -109,7 +109,7 @@ class Column:
         self,
         lowerBound: Union[Column, LiteralType, DateTimeLiteral, DecimalLiteral],
         upperBound: Union[Column, LiteralType, DateTimeLiteral, DecimalLiteral],
-    ) -> Column: ...
+    ) -> bool: ...
```

Comment:
Note for reviewer:
https://github.com/apache/spark/blob/ff1fc5ed4b685b4f5f83d5f600b24f089dd4522e/python/pyspark/sql/column.py#L801

Reply:
This doesn't look right.

Reply:
Done, : )